doc/api/api.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0
+---
+title: Metamaps
+version: v2
+baseUri: http://metamaps.cc/api/v2
+mediaType: application/json
+
+securitySchemes:
+  - oauth_2_0:
+      description: |
+        OAuth 2.0 implementation
+      type: OAuth 2.0
+      settings:
+        authorizationUri: https://metamaps.cc/api/v2/oauth/authorize
+        accessTokenUri: https://metamaps.cc/api/v2/oauth/token
+        authorizationGrants: [ authorization_code, password, client_credentials, implicit, refresh_token ]
```

Comment:
sorry about the delay. for this, we just want authorization_code and client_credentials. 

these are the defaults that doorkeeper suggests, so I went with those, as per: 
https://github.com/metamaps/metamaps/blob/develop/config/initializers/doorkeeper.rb#L83